src/app/shared/services/studio-configuration.ts

```diff
@@ -0,0 +1,12 @@
+interface BlueprintConfiguration {
+  ServerPostrollDuration: number
+}
+
+interface StudioSettings {
+  mediaPreviewUrl: string
+}
+
+export interface StudioConfiguration {
+  settings: StudioSettings
+  blueprintConfiguration: BlueprintConfiguration
+}
```

Comment:
I didn't catch this the first time around, but this file is in the wrong place.
You have placed in the `services` folder which is a folder reserved for Services, like the `ConfigurationService`.
The definitions of our interfaces and classes should be in our `models` folder.